lib/esi/calls.rb

```diff
@@ -60,6 +61,16 @@ def page=(page)
       def paginated?
         !@paginated
       end
+
+      private
+
+      def sorted_params
```

Comment:
It's a personal thing but I love one lines. Not suggesting to change it, might be less readable.
```ruby
def sorted_params 
  params.transform_values { |v| v.respond_to?(:sort) ? v.sort : v }.sort
end
```